framework/test/unit/modules/token/stores/supported_tokens.spec.ts

```diff
@@ -276,12 +276,40 @@ describe('SupportedTokensStore', () => {
 
 			await expect(store.removeSupportForToken(context, tokenID)).resolves.toBeUndefined();
 			await expect(store.has(context, Buffer.from([1, 1, 1, 1]))).resolves.toBeTrue();
+			// Checking that the store has the correct data.
 			await expect(store.get(context, Buffer.from([1, 1, 1, 1]))).resolves.toEqual({
 				supportedTokenIDs: [
 					Buffer.from([1, 1, 1, 1, 1, 0, 1, 1]),
+					tokenID,
 					Buffer.from([1, 1, 1, 1, 1, 0, 0, 1]),
 				],
 			});
 		});
```

Comment:
it would be nice to have a case where
3 tokens are supported and become 2 tokens with same order after removing

Reply:
Update from L279